en/people/floriana-ferrara.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Floriana Ferrara
+lang: en
+permalink: /en/people/floriana-ferrara.html
+layout: people
+role: Technical Project Manager
+is_new: true
+twitter_user: FlorianaFer
+medium_user: FlorianaFer
+linkedin_url: https://www.linkedin.com/in/filomena-floriana-ferrara-50588912 
+ref: floriana-ferrara
+parent_ref: team
+start_date: September 04, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 80,000
+approved_by_court: September 24, 2017
+---
+I am from Taranto, I earned a degree in Information Science at University of Bari and I am a computer expert. 
```

Comment:
"I am", "I earned", "I am" -> I'd rephrase it a bit, eg. "I'm a computer expert originally from Taranto who has earned a degree in Information Science at University of Bari"